### PR TITLE
Update MessageAttempt.cs

### PR DESCRIPTION
**Pull request by Obl3n**
## Motivation
This should solve #819 problem

## Solution

Making nullable MessageStatuse and StatusCodeClass.

### CODE REVIEW

**Review by tasn — APPROVED** [review]

**Comment by tasn** [comment]
Whoops, thanks!

**Review by tasn — APPROVED** [review]

File: csharp/Svix/MessageAttempt.cs [hunk 1]
```diff
@@ -140,8 +140,8 @@ public MessageAttemptOut GetAttempt(string appId, string attemptId, string messa
                     endpointId,
                     options?.Iterator,
                     options?.Limit,
-                    (Svix.Model.MessageStatus)options?.Status,
-                    (Svix.Model.StatusCodeClass)options?.Code,
+                   options?.Status,
```
Comment (tasn) [inline csharp/Svix/MessageAttempt.cs:143]:
```suggestion
                    options?.Status,
```
Missing spacing.

File: csharp/Svix/MessageAttempt.cs [hunk 2]
```diff
@@ -204,8 +204,8 @@ public MessageAttemptOut GetAttempt(string appId, string attemptId, string messa
                     options?.EndpointId,
                     options?.Iterator,
                     options?.Limit,
-                    (Svix.Model.MessageStatus)options?.Status,
-                    (Svix.Model.StatusCodeClass)options?.Code,
+                   options?.Status,
```
Comment (tasn) [inline csharp/Svix/MessageAttempt.cs:207]:
```suggestion
                    options?.Status,
```

**Review by svix-andor — APPROVED** [review]

**Comment by Obl3n** [comment]
@tasn done!

**Review by tasn — APPROVED** [review]

**Pull request merged**